semgrep/semgrep/output.py

```diff
@@ -97,7 +101,26 @@ def finding_to_line(
                 else:
                     line_number = f"{start_line + i}"
 
+                if per_line_max_chars_limit and len(line) > per_line_max_chars_limit:
+                    stripped = True
+                    is_first_line = i == 0
+                    if is_first_line:
+                        line = (
+                            line[
+                                start_col - 1 : start_col - 1 + per_line_max_chars_limit  # type: ignore
```

Comment:
Why is type checking ignored here?

Reply:
Oh I see. We do a dictionary get on line 82-83. No change necessary then will fix at some point in the future